internal/event/target/webhook.go

```diff
@@ -155,7 +155,10 @@ func (target *WebhookTarget) isActive() (bool, error) {
 	}
 	io.Copy(io.Discard, resp.Body)
 	resp.Body.Close()
-	// No network failure i.e response from the target means its up
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Maybe:
```suggestion
	if resp.StatusCode >= http.StatusBadRequest {
```

Only return errors when >= 400